### PR TITLE
Formatting of metatype file to keep spacing consistent

### LINKED ISSUE #18107: Fix file formatting on the transport's metatype.xml
The file dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml has inconsistent formatting due to whitespaces and `tab` characters. The file should be re-formatted to be consistent throughout it entirety.

### PR DESCRIPTION
**Pull request by mrsaldana**
Fixes #18107

### CODE REVIEW

**Comment by mrsaldana** [comment]
#build

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -333,4 +360,4 @@
         <Object ocdref="com.ibm.ws.http.log.access"/>
     </Designate>
 
-</metatype:MetaData>
+</metatype:MetaData>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml]: Can you just add the new line in at the end here?

Comment (mrsaldana) [reply dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml]: Actually, I did push from local branch with new line from line 363 to 364, and it appeared like that on the merge editor, but for some reason it isn't appearing in the GIT view

**Review by pnicolucci — APPROVED** [review]
Everything looks good now.

**Comment by mrsaldana** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#run-libby-bot

**Pull request merged**